guides/doc-Managing_Content/master.adoc

```diff
@@ -56,6 +56,8 @@ endif::[]
 
 include::common/assembly_managing-container-images.adoc[leveloffset=+1]
 
+include::common/assembly_package-mode-and-image-mode-hosts.adoc
```

Comment:
```suggestion
include::common/assembly_package-mode-and-image-mode-hosts.adoc[leveloffset=+1]
```
I'm surprised the build doesn't fail on this.

Reply:
I deleted it. It shouldn't be in the Managing Content guide.